### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by dzianisreznik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ngmp2018q4",
+  "version": "1.0.0",
+  "description": "Epam NodeJS Global Mentoring Program Q4",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon --exec babel-node app.js",
```
Comment (AlexeyBarkow) [inline package.json]: src/app.js
otherwise it fails

Comment (AlexeyBarkow) [inline package.json]: Currently nodemon restarts app on any change in repository. It's a good practice to add nodemon config in order to make nodemon watch changes only inside src folder

Comment (dzianisreznik) [reply package.json]: 😓 Forgot to fix after changing the project structure.

**Comment by AlexeyBarkow** [comment]
Great work, I think this PR now can be merged

**Pull request merged**